src/litdata/processing/functions.py

```diff
@@ -209,6 +209,7 @@ def map(
     reader: Optional[BaseReader] = None,
     batch_size: Optional[int] = None,
     start_method: Optional[str] = None,
+    optimize_dns: Optional[bool] = None,
```

Comment:
Just out of curiosity, is there a reason for not setting it to False?

Reply:
Changed the default to False